lib/kubeclient/entity_list.rb

```diff
@@ -5,12 +5,17 @@ module Common
     class EntityList < DelegateClass(Array)
       attr_reader :kind, :resourceVersion
 
-      def initialize(kind, resource_version, list)
+      def initialize(kind, resource_version, list, continue = nil)
         @kind = kind
         # rubocop:disable Style/VariableName
         @resourceVersion = resource_version
+        @continue = continue
         super(list)
       end
+
+      def last?
```

Comment:
Nice!  I think this is worth keeping if we return to low-level version.

Reply:
Yeah absolutely, this made the calling code a lot cleaner